app/user/validations.js

```diff
@@ -0,0 +1,8 @@
+import { validatePresence, validateLength } from 'ember-changeset-validations/validators';
```

Comment:
Yes, and you've got to give credit to the guy(s) behind `ember-bootstrap` for making their forms integrate so seamlessly with it too 😁